web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -359,6 +359,13 @@ const EXTERNAL_SERVICES = [
     endpoint: 'https://neurosift.app?p=/nwb&url=$asset_dandi_url$&st=lindi&dandisetId=$dandiset_id$&dandisetVersion=$dandiset_version$', // eslint-disable-line max-len
   },
 
+  {
+    name: 'Neurosift',
+    regex: /\.nwb.lindi.tar$/,
+    maxsize: Infinity,
+    endpoint: 'https://neurosift.app?p=/nwb&url=$asset_dandi_url$&st=lindi&dandisetId=$dandiset_id$&dandisetVersion=$dandiset_version$', // eslint-disable-line max-len
+  },
```

Comment:
Since this is exactly the same as the previous Neurosift spec but for the file extension, would it make sense to simply update the regex in line 357 to accept both `.nwb.lindi.json` and `.nwb.lindi.tar`?

Reply:
Made the change.